### ISSUE TITLE
Sulu 2.4 throws error because of missing restoreFormMapping if the SuluTrashBundle is not registered

### ISSUE THREAD

**Issue by TheCadien** [sulu/sulu#6275]
| Q | A
| --- | ---
| Sulu 2.4@Beta | 
| 8.0 | 
| SQL 8 |

# Actual Behavior

If the SuluTrashBundle isn't installed in bundles.php after Update, it throws the following error

```
index.js:9 Uncaught (in promise) TypeError: Cannot read property 'restoreFormMapping' of undefined
    at index.js:9
    at initializer.js:119
    at Array.forEach (<anonymous>)
    at initializer.js:118
```

**Comment by niklasnatter** [comment]
Will be fixed in #6248 🙂

**Comment by niklasnatter** [comment]
Fixed in #6248
